utils/parser.py

```diff
@@ -7,9 +7,8 @@
 from db.config import settings
 from db.models import Streams, TVStreams
 from db.schemas import Stream, UserData
-from streaming_providers.realdebrid.utils import (
-    order_streams_by_instant_availability_and_date,
-)
+import streaming_providers.realdebrid.utils as RealDebridUtils
+import streaming_providers.debridlink.utils as DebridLinkUtils
```

Comment:
```suggestion
from streaming_providers.realdebrid import utils as rd_utils
from streaming_providers.debridlink import utils as dl_utils
```

Reply:
I don't have much development experience with Python. Curious what is the difference between those import ways?

Reply:
in the above imports statements are doing the same. Importing utils namespace into current file. The `from` statement is specially used to specifying that is importing from submodule contents. Again, this is something I've been practiced from pythonic world and my preference. 

The main reason for the changes was to follow python variable guide as snake case. Such as `RealDebridUtils` -> `rd_utils`. `rd` is short name.